tests/test_protocol.py

```diff
@@ -1,18 +1,101 @@
 from pathlib import Path
 
+import numpy as np
 import pytest
+import xarray as xa
 
-from openlifu import Protocol
+from openlifu import Protocol, Transducer
+from openlifu.db import Session
 
 
 @pytest.fixture()
 def example_protocol() -> Protocol:
     return Protocol.from_file(Path(__file__).parent/'resources/example_db/protocols/example_protocol/example_protocol.json')
 
+@pytest.fixture()
+def example_transducer() -> Transducer:
+    return Transducer.from_file(Path(__file__).parent/"resources/example_db/transducers/example_transducer/example_transducer.json")
+
+@pytest.fixture()
+def example_session() -> Session:
+    return Session.from_file(Path(__file__).parent/"resources/example_db/subjects/example_subject/sessions/example_session/example_session.json")
+
+@pytest.fixture()
+def example_transducer_transform() -> np.ndarray:
+    return np.array(
+        [
+            [-1,    0,	0,	0],
+            [0,	0.05,	0.998749217771909,	-105],
+            [0,	0.998749217771909,	-0.05,	5],
+            [0,	0,	0,	1]
+        ]
+    )
+
+@pytest.fixture()
+def example_volume() -> xa.Dataset:
```

Comment:
Having an example_volume fixture that returns `None` is not very nice. It also doesn't satisfy the type annotation. If we want to pass `None` for the volume, then we should directly pass `None` in `test_calc_solution` rather than pretending that `None` is a volume. Later if we add an example volume we can create a new test that tests `calc_solution` with that volume. It is okay to leave the commented out code here, just comment out the entire thing so that the `example_volume` test fixture remains undefined for now.

Reply:
done